linux/open_vm_tools/install_ovt_from_package.yml

```diff
@@ -9,6 +9,13 @@
   include_tasks: ../utils/config_repos.yml
   when: guest_os_ansible_system == "linux"
 
+- name: "Reconfigure xmlsec1-devel on {{ vm_guest_os_distribution }}"
```

Comment:
In your cases, this package was not installed and here is to install it. So please change `Reconfigure` to `Install`.